docs/wiki/00_SERVER_INSTALL_AND_CONFIGURATION/01_LORIS_Install/CentOS/README.md

```diff
@@ -37,7 +37,11 @@ sudo yum install epel-release
 sudo yum install http://rpms.remirepo.net/enterprise/remi-release-7.rpm
 sudo yum install yum-utils
 sudo yum update
-sudo yum --enablerepo=remi-php73 install php php-pdo php-pdo_mysql php73-php-fpm php73-php-gd php73-php-json php73-php-mbstring php73-php-mysqlnd php73-php-xml php73-php-xmlrpc php73-php-opcache php73-php-mysql
+
+# By default, the repository for PHP 5.4 is enabled
+sudo yum-config-manager --disable remi-php54
+sudo yum-config-manager --enable remi-php73
```

Comment:
This should be using PHP 7.4, not 7.3. 7.3 is end of life in 5 months.

Reply:
ok -- this update should then also be made in the main Readme and the docs, and possibly re-test the installs.

Reply:
Readme updated. This need to be tested.

Reply:
great, thanks @laemtl - pls test when you have a moment.

Reply:
Replace 7.3 in the Ubuntu install file too.